packages/block-library/src/buttons/index.js

```diff
@@ -23,6 +23,7 @@ export const settings = {
 	),
 	icon,
 	keywords: [ __( 'link' ) ],
+	example: {},
```

Comment:
maybe we can be more creative and use something close to the two or three buttons pattern here.